test/e2e/predictor/test_torchserve.py

```diff
@@ -39,34 +38,33 @@ def test_torchserve_kserve():
     predictor = V1beta1PredictorSpec(
         min_replicas=1,
         pytorch=V1beta1TorchServeSpec(
+            image="pytorch/torchserve-kfs:0.8.2",
```

Comment:
Do we still need this override as it is already configured in the kustomize.yaml ?